local-volume/provisioner/pkg/discovery/discovery.go

```diff
@@ -179,25 +179,24 @@ func (d *Discoverer) createPV(file, class string, config common.MountConfig, cap
 	glog.Infof("Created PV %q for volume at %q", pvName, outsidePath)
 }
 
-// roundDown the capacity so that it is easier to read
+// roundDown the capacity to an easy to read value
 func roundDownCapacityPretty(capacity int64) int64 {
-	// capacity is exactly divisible by Gi
-	if capacity%1024*1024*1024 == 0 {
-		return capacity
-	}
-	sizeGi := util.RoundUpSize(capacity, 1024*1024*1024)
-	if (sizeGi - 1) >= 10 {
-		// rounddown to nearest Gi
-		return (sizeGi - 1) * 1024 * 1024 * 1024
-	}
-	// capacity is exactly divisible by Mi
-	if capacity%1024*1024 == 0 {
-		return capacity
-	}
-	sizeMi := util.RoundUpSize(capacity, 1024*1024)
-	if (sizeMi - 1) >= 10 {
-		// rounddown to nearest Mi
-		return (sizeMi - 1) * 1024 * 1024
+
+	const (
```

Comment:
You can put this in one of the common files

Reply:
Done. I put it with the RoundUpSize & RoundDownSize methods.